src/main/java/org/spongepowered/common/mixin/api/mcp/entity/EntityMixin_API.java

```diff
@@ -644,16 +641,26 @@ public EntityArchetype createArchetype() {
     }
 
     @Override
-    public org.spongepowered.api.data.value.Value.Mutable<Boolean> gravity() {
-        return this.getValue(Keys.HAS_GRAVITY).get();
+    public Set<Value.Immutable<?>> getValues() {
```

Comment:
Throw us a TODO here to merge custom and vanilla values and then return the result.